### PR TITLE
Fix Target the same element with multiple jQuery Selectors

### LINKED ISSUE #8438: Instructions not too clear
Challenge [Target the same element with multiple jQuery Selectors](https://www.freecodecamp.com/challenges/target-the-same-element-with-multiple-jquery-selectors#?solution=%0Afccss%0A%20%20%24(document).ready(function()%20%7B%0A%20%20%20%20%24(%22button%22).addClass(%22animated%22)%3B%0A%20%20%20%20%24(%22.btn%22).addClass(%22shake%22)%3B%0A%20%20%20%20%24(%22%23target1%22).addClass(%22btn-primary%22)%3B%0A%20%20%7D)%3B%0Afcces%0A%0A%3C!--%20Only%20change%20code%20above%20this%20line.%20--%3E%0A%0A%3Cdiv%20class%3D%22container-fluid%22%3E%0A%20%20%3Ch3%20class%3D%22text-primary%20text-center%22%3EjQuery%20Playground%3C%2Fh3%3E%0A%20%20%3Cdiv%20class%3D%22row%22%3E%0A%20%20%20%20%3Cdiv%20class%3D%22col-xs-6%22%3E%0A%20%20%20%20%20%20%3Ch4%3E%23left-well%3C%2Fh4%3E%0A%20%20%20%20%20%20%3Cdiv%20class%3D%22well%22%20id%3D%22left-well%22%3E%0A%20%20%20%20%20%20%20%20%3Cbutton%20class%3D%22btn%20btn-default%20target%22%20id%3D%22target1%22%3E%23target1%3C%2Fbutton%3E%0A%20%20%20%20%20%20%20%20%3Cbutton%20class%3D%22btn%20btn-default%20target%22%20id%3D%22target2%22%3E%23target2%3C%2Fbutton%3E%0A%20%20%20%20%20%20%20%20%3Cbutton%20class%3D%22btn%20btn-default%20target%22%20id%3D%22target3%22%3E%23target3%3C%2Fbutton%3E%0A%20%20%20%20%20%20%3C%2Fdiv%3E%0A%20%20%20%20%3C%2Fdiv%3E%0A%20%20%20%20%3Cdiv%20class%3D%22col-xs-6%22%3E%0A%20%20%20%20%20%20%3Ch4%3E%23right-well%3C%2Fh4%3E%0A%20%20%20%20%20%20%3Cdiv%20class%3D%22well%22%20id%3D%22right-well%22%3E%0A%20%20%20%20%20%20%20%20%3Cbutton%20class%3D%22btn%20btn-default%20target%22%20id%3D%22target4%22%3E%23target4%3C%2Fbutton%3E%0A%20%20%20%20%20%20%20%20%3Cbutton%20class%3D%22btn%20btn-default%20target%22%20id%3D%22target5%22%3E%23target5%3C%2Fbutton%3E%0A%20%20%20%20%20%20%20%20%3Cbutton%20class%3D%22btn%20btn-default%20target%22%20id%3D%22target6%22%3E%23target6%3C%2Fbutton%3E%0A%20%20%20%20%20%20%3C%2Fdiv%3E%0A%20%20%20%20%3C%2Fdiv%3E%0A%20%20%3C%2Fdiv%3E%0A%3C%2Fdiv%3E%0A) has an issue.
User Agent is: <code>Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/50.0.2661.94 Safari/537.36</code>.
Please describe how to reproduce this issue, and include links to screenshots if possible.

My code:
```html

<script>
  $(document).ready(function() {
    $("button").addClass("animated");
    $(".btn").addClass("shake");
    $("#target1").addClass("btn-primary");
  });
</script>

<!-- Only change code above this line. -->

<div class="container-fluid">
  <h3 class="text-primary text-center">jQuery Playground</h3>
  <div class="row">
    <div class="col-xs-6">
      <h4>#left-well</h4>
      <div class="well" id="left-well">
        <button class="btn btn-default target" id="target1">#target1</button>
        <button class="btn btn-default target" id="target2">#target2</button>
        <button class="btn btn-default target" id="target3">#target3</button>
      </div>
    </div>
    <div class="col-xs-6">
      <h4>#right-well</h4>
      <div class="well" id="right-well">
        <button class="btn btn-default target" id="target4">#target4</button>
        <button class="btn btn-default target" id="target5">#target5</button>
        <button class="btn btn-default target" id="target6">#target6</button>
      </div>
    </div>
  </div>
</div>

```
Please clear up instructions and explain better for student sto understand as it took some time figuring this challenge out.

### PR DESCRIPTION
**Pull request by keevcodes**
#### Pre-Submission Checklist

- [x] Your pull request targets the `staging` branch of FreeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/git-rebase#squashing-multiple-commits-into-one) them into one commit).
- [x] All new and existing tests pass the command `npm run test-challenges`. Use `git commit --amend` to amend any fixes.
#### Type of Change

- [x] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)
#### Checklist:

- [x] Tested changes locally.
- [x] Closes currently open issue (replace XXXX with an issue no): Closes #8438 
#### Description

I added changes to the instructions for the jquery/json challenge. Users are now given a more detailed description that the exercise asks them to write three different ways to add a class. 

Instructions are more clear to users that they must use three different ways to add a class

### CODE REVIEW

File: seed/challenges/01-front-end-development-certification/jquery.json [hunk 1]
```diff
@@ -344,10 +344,12 @@
       "title": "Target the same element with multiple jQuery Selectors",
       "description": [
         "Now you know three ways of targeting elements: by type: <code>$(\"button\")</code>, by class: <code>$(\".btn\")</code>, and by id <code>$(\"#target1\")</code>.",
+        "Although it is possbile to add multiple classes in a single <code>.addClass()</code> call, let's add them to the same element in <em>three sperate ways.",
```
Comment (erictleung) [inline seed/challenges/01-front-end-development-certification/jquery.json]: You're missing the matching `<em>` tag at the end of this line. Which might be why Travis is giving you issues.

**Comment by erictleung** [comment]
@andmckvr13 thanks for your contribution! Some things to change. Up above, where is says `Closes #XXXX`, please change this to `Closes #8438` so that it references the issue you're fixing and will automatically close the issue after you PR is accepted. I've also commented on the line that is probably causing Travis to fail a check.

File: seed/challenges/01-front-end-development-certification/jquery.json [hunk 2]
```diff
@@ -344,10 +344,12 @@
       "title": "Target the same element with multiple jQuery Selectors",
       "description": [
         "Now you know three ways of targeting elements: by type: <code>$(\"button\")</code>, by class: <code>$(\".btn\")</code>, and by id <code>$(\"#target1\")</code>.",
+        "Although it is possbile to add multiple classes in a single <code>.addClass()</code> call, let's add them to the same element in <em>three sperate ways</em>.",
         "Using <code>.addClass()</code>, add only one class at a time to the same element, three different ways:",
         "Add the <code>animated</code> class to all elements with type <code>button</code>.",
         "Add the <code>shake</code> class to all the buttons with class <code>.btn</code>.",
         "Add the <code>btn-primary</code> class to the button with id <code>#target1</code>."
```
Comment (erictleung) [inline seed/challenges/01-front-end-development-certification/jquery.json]: You also need a comma at the end of this line 351

```
...button with id <code>#target1</code>.",
```

**Comment by erictleung** [comment]
@andmckvr13 thanks for being quick updating your PR! Looks like everything looks good for @camperbot and Travis. One more thing: can you [squash your commits to one](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/git-rebase#squashing-multiple-commits-into-one)? Thanks! Let us know if you have any issues squashing.

**Comment by andmckvr13** [comment]
You're welcome Eric! I'm very excited for this first contribution, all be
it a small one. I must be off to work now, but I will take a look at the
squashing issue later.

**Comment by andmckvr13** [comment]
I tired git rebase -i HEAD~n and received this message "fatal: needed a
single revision, invalid upstream HEAD

**Comment by erictleung** [comment]
@andmckvr13 the `n` should be a number. In your case it should be `3`. So the command you want it:

```
git rebase -i HEAD~3
```

As the instructions say, you'll need to leave the top one alone and make changes to the other two lines.

**Comment by erictleung** [comment]
@andmckvr13 also, no need to down play your contribution. No change is too small to not be proud of 😃

**Comment by andmckvr13** [comment]
Thank you for the help Eric.

**Comment by andmckvr13** [comment]
Greetings from Hamburg Eric,


I received a pop up notification from my NotePad
pick 34aad3d added more descriptive instructions for users. Clearer that
they must use three different methods to add a class
pick 88b76e6 Improved instructions for users to see that they must use
three different ways to add a class
pick b00f9c0 Added clearer instructions to JQuery/Json challenge

# Rebase d0afae8..b00f9c0 onto d0afae8 (3 command(s))
#
# Commands:
# p, pick = use commit
# r, reword = use commit, but edit the commit message
# e, edit = use commit, but stop for amending
# s, squash = use commit, but meld into previous commit
# f, fixup = like "squash", but discard this commit's log message
# x, exec = run command (the rest of the line) using shell
# d, drop = remove commit
#
# These lines can be re-ordered; they are executed from top to bottom.
#
# If you remove a line here THAT COMMIT WILL BE LOST.
#
# However, if you remove everything, the rebase will be aborted.


not exactly sure what this would like me to do, hopefully the squash worked

**Comment by erictleung** [comment]
@ You'll need to leave the top commit alone, then change the other two `pick` to squash so it should look something like this

```
pick 34aad3d added more descriptive instructions for users. Clearer that they must use three different methods to add a class
squash 88b76e6 Improved instructions for users to see that they must use three different ways to add a class
squash b00f9c0 Added clearer instructions to JQuery/Json challenge
```

Save it and exist and you might need to go through one more screen but after than it should be good.

**Comment by andmckvr13** [comment]
Thanks so much for the guidance. I believed I solved it. I will also look
for resources to learn more about squashing/git so that I don't bother you
in the future.

**Comment by erictleung** [comment]
@andmckvr13 no worries. Here's a link that might help https://robots.thoughtbot.com/git-interactive-rebase-squash-amend-rewriting-history.

I still don't see the squash up on GitHub. You'll still need to push your update to GitHub so

```shell
git push origin fix/instructionsnottooclear --force
```

**Comment by andmckvr13** [comment]
Thanks for the link. I've pushed the squash hopefully

**Comment by erictleung** [comment]
@andmckvr13 great! Looks like you've squashed them 👍  Unfortunately, it looks like you've missed escaping one quotation.

image

Also, it looks like you've misspelled `possible` and you're missing the matching `<em>` tag.

Here's some commands to help you out so that you don't have to squash things again after you make your changes.

```
git checkout fix/instructionsnottooclear
...# make your edits here
git commit --amend
git push origin fix/instructionsnottooclear --force
```

Amending commits just use your previous commit but just add on new edits. Let me know if you have any issues or questions about it.

File: seed/challenges/01-front-end-development-certification/jquery.json [hunk 3]
```diff
@@ -344,10 +344,13 @@
       "title": "Target the same element with multiple jQuery Selectors",
       "description": [
         "Now you know three ways of targeting elements: by type: <code>$(\"button\")</code>, by class: <code>$(\".btn\")</code>, and by id <code>$(\"#target1\")</code>.",
+        "Although it is possbile to add multiple classes in a single <code>.addClass()</code> call, let's add them to the same element in <em>three sperate ways</em>.",
+        "Using <code>.addClass()</code>, add only one class at a time to the same element, three different ways:",
         "Using <code>.addClass()</code>, add only one class at a time to the same element, three different ways:",
         "Add the <code>animated</code> class to all elements with type <code>button</code>.",
         "Add the <code>shake</code> class to all the buttons with class <code>.btn</code>.",
         "Add the <code>btn-primary</code> class to the button with id <code>#target1</code>."
```
Comment (raisedadead) [inline seed/challenges/01-front-end-development-certification/jquery.json]:
```diff
-"Add the <code>btn-primary</code> class to the button with id <code>#target1</code>."
+"Add the <code>btn-primary</code> class to the button with id <code>#target1</code>.",
```

You need a `,` at the end of this line.

**Comment by raisedadead** [comment]
@andmckvr13 please execute `npm run test-challenges` to avoid Travis CI errors, before pushing your changes. And thanks a lot for your contribution, looking forward to get this merged soon.

**Comment by andmckvr13** [comment]
Thanks everyone for the great help! Let me know if there is anything I did
wrong or need to fix.

File: seed/challenges/01-front-end-development-certification/jquery.json [hunk 4]
```diff
@@ -344,10 +344,13 @@
       "title": "Target the same element with multiple jQuery Selectors",
       "description": [
         "Now you know three ways of targeting elements: by type: <code>$(\"button\")</code>, by class: <code>$(\".btn\")</code>, and by id <code>$(\"#target1\")</code>.",
+        "Although it is possbile to add multiple classes in a single <code>.addClass()</code> call, let's add them to the same element in <em>three sperate ways</em>.",
+        "Using <code>.addClass()</code>, add only one class at a time to the same element, three different ways:",
```
Comment (raisedadead) [inline seed/challenges/01-front-end-development-certification/jquery.json]: Remove this line, this is a duplicate addition of the same instruction.

File: seed/challenges/01-front-end-development-certification/jquery.json [hunk 5]
```diff
@@ -344,10 +344,13 @@
       "title": "Target the same element with multiple jQuery Selectors",
       "description": [
         "Now you know three ways of targeting elements: by type: <code>$(\"button\")</code>, by class: <code>$(\".btn\")</code>, and by id <code>$(\"#target1\")</code>.",
+        "Although it is possbile to add multiple classes in a single <code>.addClass()</code> call, let's add them to the same element in <em>three sperate ways</em>.",
```
Comment (raisedadead) [inline seed/challenges/01-front-end-development-certification/jquery.json]: typo `sperate` > `separate`

**Comment by raisedadead** [comment]
@andmckvr13 please check the comments above, some more fixes needed.

After making the changes push the changes by following

1. `git add --all`
2. `git commit --amend --no-edit`
3. `git push origin fix/instructionsnottooclear`

Reach out to the [Contributors Chat room](https://gitter.im/FreeCodeCamp/HelpContributors), if you need assistance.

File: seed/challenges/01-front-end-development-certification/jquery.json [hunk 6]
```diff
@@ -344,10 +344,12 @@
...
       "description": [
         "Now you know three ways of targeting elements: by type: <code>$(\"button\")</code>, by class: <code>$(\".btn\")</code>, and by id <code>$(\"#target1\")</code>.",
+        "Although it is possbile to add multiple classes in a single <code>.addClass()</code> call, let's add them to the same element in <em>three separate ways</em>.",
         "Using <code>.addClass()</code>, add only one class at a time to the same element, three different ways:",
         "Add the <code>animated</code> class to all elements with type <code>button</code>.",
         "Add the <code>shake</code> class to all the buttons with class <code>.btn</code>.",
-        "Add the <code>btn-primary</code> class to the button with id <code>#target1</code>."
+        "Add the <code>btn-primary</code> class to the button with id <code>#target1</code>.",
+        "<strong>Note</strong><br>You should only be targeting one element and adding only one class at a time. Altogether, your three individual selectors will end up adding the three classes <code>shake</code>, <code>animated</code>, and <code>btn-primary</code> to <code>#target1</code>."
```
Comment (atjonathan) [inline seed/challenges/01-front-end-development-certification/jquery.json]:
```diff
- "<strong>Note</strong><br>...
+ "<strong>Note:</strong><br>...
```

Comment (erictleung) [inline seed/challenges/01-front-end-development-certification/jquery.json]: @atjonathan I don't think we generally have colons next to Notes. It is kind of a "header". I remember adding a colon next to "Note" but @SaintPeter correctly me 😃

File: seed/challenges/01-front-end-development-certification/jquery.json [hunk 7]
```diff
@@ -344,10 +344,12 @@
       "title": "Target the same element with multiple jQuery Selectors",
       "description": [
         "Now you know three ways of targeting elements: by type: <code>$(\"button\")</code>, by class: <code>$(\".btn\")</code>, and by id <code>$(\"#target1\")</code>.",
+        "Although it is possbile to add multiple classes in a single <code>.addClass()</code> call, let's add them to the same element in <em>three separate ways</em>.",
```
Comment (erictleung) [inline seed/challenges/01-front-end-development-certification/jquery.json]: There's a typo "possbile" here that needs to be "possible"

**Comment by erictleung** [comment]
@andmckvr13 just one small typo to fix and then you're good to go!

The git commands you need to fix this aren't too bad

```sh
git checkout fix/instructionsnottooclear
cd FreeCodeCamp/seed/challenges/01-front-end-development-certification # wherever your directory is
... # make your change here
git add jquery.json
git commit --amend
git push origin fix/instructionsnottooclear --force
```

Let me know on Gitter or here if you need any help.

**Comment by atjonathan** [comment]
#### **LGTM :+1:**
Thanks for contributing :tada:

**Comment by raisedadead** [comment]
@andmckvr13 Thanks a lot for this PR and being so patient and addressing all the comments and feed backs, your efforts are much appreciated.

And on that :sparkles: **congratulations** :sparkles: on your first contribution to Open Source and Free Code Camp!

Thanks.

**Comment by andmckvr13** [comment]
Thank you everyone so much for all the help! I really learned a lot from this! I look forward to more contributions

**Pull request merged**